dd-sdk-android/src/main/kotlin/com/datadog/android/rum/tracking/WhitelistPredicate.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.tracking
+
+/**
+ * Used to whitelist an Activity or Fragment in order to be considered as a RUM View Event in our
+ * Activity or Fragment view tracking strategies.
+ * @see FragmentViewTrackingStrategy
+ * @see ActivityViewTrackingStrategy
+ */
+interface WhitelistPredicate<T> {
+    /**
+     * Whether to whitelist or not a specific Component.
+     * @param component the Component that needs to be marked or not as whitelisted.
+     * It can be either a Fragment or Activity.
+     * @return true if we want this Component to be tracked, false otherwise.
```

Comment:
```suggestion
    * Decides whether the provided component should be considered as a RUM View.
    * @param component a component whose state changed
    * @return true if you want the provided component to be tracked as a View, false otherwise
```